File: unit-3/sprint-2/evaluation/firstapp/app/src/main/java/com/example/firstapp/itemViewHolder.java
package com.example.firstapp;

import android.view.View;
import android.widget.ImageView;
import android.widget.TextView;

import androidx.annotation.NonNull;
import androidx.recyclerview.widget.RecyclerView;

import com.bumptech.glide.Glide;

public class itemViewHolder extends RecyclerView.ViewHolder {
    private ImageView mIvitemImage;
    private TextView mTvitemTitle, mTvitemSUbTitle;

    public itemViewHolder(@NonNull View itemView) {
        super(itemView);
        mTvitemSUbTitle = itemView.findViewById(R.id.tvitemTitle);
        mTvitemTitle = itemView.findViewById(R.id.tvitemSubTitle);
        mIvitemImage = itemView.findViewById(R.id.iv_itemImage);
    }

    public void setData(ResponseModel responseModel){
        Glide.with(mIvitemImage).load(responseModel.getImage()).into(mIvitemImage);
        mTvitemTitle.setText(responseModel.getSubTitle());
        mTvitemSUbTitle.setText(responseModel.getTitle());
    }

}

File: unit-4/sprint-2/evaluation/unit4_codingeval_1/app/src/main/java/com/example/unit4_codingeval_1/recyclerview/ItemViewHolder.kt
package com.example.unit4_codingeval_1.recyclerview

import android.view.View
import androidx.recyclerview.widget.RecyclerView
import kotlinx.android.synthetic.main.fragment_search_item.view.*
import kotlinx.android.synthetic.main.item_layout.view.*

class ItemViewHolder(itemView: View): RecyclerView.ViewHolder(itemView) {

    fun setData(dataModel: ModelClass){
        itemView.apply {
            tvShowName.text = dataModel.name
            tvShowPrise.text = dataModel.prise

        }
    }
}